pilot/pkg/model/config.go

```diff
@@ -45,6 +45,40 @@ const (
 	RevisionLabel = "istio.io/rev"
 )
 
+// ConfigKey describe a specific config item depends on or get changed.
+// In most cases name is config name except that for ServiceEntry it's Service.Hostname.
```

Comment:
what does this mean?

Reply:
In most cases, the name is the config name. However, for ServiceEntry it is service's FQDN.

Reply:
We use `model.Service.Hostname` rather that `ServiceEntry.Name` here. For other config types this name is just `model.Config.Name`.